Assignment2/merysan1999-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,73 @@
+{
+    "@context": {
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+		"subjects": "http://www.w3.org/Ej2b/Subjects/",
+        "teachers": "http://www.w3.org/Ej2b/Teachers/",
+        "name": "http://www.w3.org/Ej2b/Name/",
+        "description": "http://www.w3.org/Ej2b/Description/",
```

Comment:
You don't need / at the end of teachers, name, description. You have to include it or a # when defining a prefix not for attribute definitions. 
You don't need to change them, but remember to provide good definitions for the future.